exporter/collector/logs_test.go

```diff
@@ -146,17 +165,24 @@ func TestLogMapping(t *testing.T) {
 			mr: func() *monitoredrespb.MonitoredResource {
 				return nil
 			},
-			expectedEntries: []logging.Entry{
+			expectedEntries: []*logpb.LogEntry{
 				{
-					Payload:   []byte(`{"message": "hello!"}`),
-					Timestamp: testObservedTime,
-					HTTPRequest: &logging.HTTPRequest{
-						Request:                        makeExpectedHTTPReq("GET", "https://www.example.com", "https://www.example2.com", "test", nil),
+					LogName:   logName,
+					Timestamp: timestamppb.New(testObservedTime),
+					Payload: &logpb.LogEntry_JsonPayload{JsonPayload: &structpb.Struct{Fields: map[string]*structpb.Value{
+						"message": {Kind: &structpb.Value_StringValue{StringValue: "hello!"}},
+					}}},
+					HttpRequest: &logtypepb.HttpRequest{
+						RequestMethod:                  "GET",
+						UserAgent:                      "test",
+						Referer:                        "https://www.example2.com",
+						RequestUrl:                     "https://www.example.com",
+						Protocol:                       "HTTP/1.1",
 						RequestSize:                    1,
 						Status:                         200,
 						ResponseSize:                   1,
-						LocalIP:                        "192.168.0.2",
```

Comment:
Seems weird that there is no `LocalIP` in the plane protobufs.

Reply:
See https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/pull/697/files#diff-5b79131adcc4992f4a98bfaa91757bb5cc73c66f4412ed468d7e2fcd4fa7640eL600.  LocalIP == ServerIp

Reply:
Got it hanks